app/models/case/base.rb

```diff
@@ -97,7 +97,13 @@ def self.searchable_document_tsvector
   scope :overturned_ico, -> { where(type: ['Case::OverturnedICO::FOI',
                                            'Case::OverturnedICO::SAR'])}
 
+  # TODO - This is really confusing naming - one is a SAR that's not Offender
+  # The other is a Case that's not an Offender SAR
+  # Change the name of the first one to :sar_non_offender to make it clearer
   scope :non_offender_sar, -> { where(type: 'Case::SAR::Standard') }
+  scope :not_offender_sar, -> { where.not(type: ['Case::SAR::Offender'] ) }
+
+  scope :default_for_non_responders, -> { not_offender_sar }
```

Comment:
👍  better name especially make it more logical sense in the case_finder_service.rb file